### PR TITLE
fix fatty being restricted by no-air-restriction

### PR DESCRIPTION
**Pull request by PhilipJFryFAF**

### CODE REVIEW

**Comment by PhilipJFryFAF** [comment]
not 100% sure if this is the best way to fix the issue maybe putting a unique category for the fatty in the bp file and excluding him like that would be better...

**Comment by PhilipJFryFAF** [comment]
Never mind just double checked and without the PR the no-air restriction affects 106 units and with the PR it affects 105 (the one difference being the Fatboy).

File: lua/ui/lobby/UnitsRestrictions.lua [hunk 1]
```diff
@@ -78,7 +78,7 @@ Expressions = {
     -- added exclusion of engineers and structures because they are restricted by other presets
     LAND        = "(LAND - ENGINEER - STRUCTURE)",
     -- added restriction of AA structures because they are not needed when all air units are restricted
-    AIR         = "((STRUCTURE * (ANTIAIR + AIRSTAGINGPLATFORM)) + (AIR - POD - SATELLITE) + (LAND * ANTIAIR - DIRECTFIRE) + (TECH3 * NAVAL * CARRIER * AIRSTAGINGPLATFORM))",
+    AIR         = "((STRUCTURE * (ANTIAIR + AIRSTAGINGPLATFORM)) + (AIR - POD - SATELLITE) + (LAND * ANTIAIR - DIRECTFIRE - INDIRECTFIRE) + (TECH3 * NAVAL * CARRIER * AIRSTAGINGPLATFORM))",
```
Comment (HUSSAR-mtrela) [inline lua/ui/lobby/UnitsRestrictions.lua]: nice try of fixing it but there is actually a better way of fixing it and I will submit PR that also fixes some other issues mentioned in #1981

**Comment by MrNukealizer** [comment]
It still seems like a good change in case any mods add units with ANTIAIR and INDIRECTFIRE but not DIRECTFIRE.

**Comment by HUSSAR-mtrela** [comment]
@aeoncleanse this PR can be close because I have better fix in PR #1990

**Comment by PhilipJFryFAF** [comment]
thanks Hussar i'll close the PR

**Pull request closed without merging**